staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -572,6 +572,9 @@ const (
 
 	// Summarizes that all the DataVolumes attached to the VMI are Ready or not
 	VirtualMachineInstanceDataVolumesReady VirtualMachineInstanceConditionType = "DataVolumesReady"
+
+	// Indicates whether the VMI is live migratable
+	VirtualMachineInstanceIsStorageLiveMigratable VirtualMachineInstanceConditionType = "VMIStorageLiveMigratable"
```

Comment:
Why the `VMI` prefix?

Reply:
Because it is a VMI condition, should I remove it?

Reply:
I dunno, does it seem weird that there are other VMI conditions that don't have the prefix?  This is the only one

Reply:
You are right, I removed it